doc/developers/minimal_reproducer.rst

```diff
@@ -199,21 +199,21 @@ As already mentioned, the key to communication is the readability of the code
 and good formatting can really be a plus. Notice that in the previous snippet
 we:
 
-    - try to limit all lines to a maximum of 79 characters to avoid horizontal
-      scrollbars in the code snippets blocks rendered on the GitHub issue;
-    - use blank lines to separate groups of related functions;
-    - place all the imports in their own group at the beginning.
+- try to limit all lines to a maximum of 79 characters to avoid horizontal
+    scrollbars in the code snippets blocks rendered on the GitHub issue;
```

Comment:
```suggestion
- try to limit all lines to a maximum of 79 characters to avoid horizontal
  scrollbars in the code snippets blocks rendered on the GitHub issue;
```